karma.conf.js

```diff
@@ -52,9 +52,35 @@ module.exports = function(config) {
     // preprocess matching files before serving them to the browser
     // available preprocessors: https://npmjs.org/browse/keyword/karma-preprocessor
     preprocessors: {
+		'src/index.js': ['webpack'],
     },
 
 
+	webpack: {
```

Comment:
using karma-webpack since the file built with `npm run build` is now a commonjs output so it doesn't work in the browser (and we can't use it in the tests).